### ISSUE TITLE
Unable to connect to remote server

### ISSUE THREAD

**Issue by inzi2u** [ABJC/ABJC-tvOS#45]
### Description of the Issue

i am unable to log in to my jellyfin server. when pressing continue after entering the details, i get an authfailed.
i know the password and username are correct.
I would like to know what exactly needs to be entered in the 3 spaces on the server selection screen.
My server is a remote server behind a nginx reverse proxy .
Domain https://jellyfin.xxxxx.yyyy connects to the webui fine when i try via a browser

what i entered in the fields were

jellyfin
443
https://jellyfin.xxxxx.yyyy
https On
still get an auth failed

### tvOS Version

15

### ABJC Version

1.0.0 (28)

### ABJC TestFlight Build

28

**Comment by beefycheddar** [comment]
I am having the same issue. The one difference is that I have jellyfin configured as a subfolder, instead of subdomain. Any insight you can provide will be much appreciated.

**Comment by inzi2u** [comment]
To be honest, I still have not figured out why it would work.
Waiting on the dev to say something.

Regards

**Comment by noahkamara** [comment]
Hi There,
could you email me the mail-address you've used to sign into TestFlight? I'd like to add you to the alpha build, which should solve this but isn't quite ready for general users yet.

My Email is: [mailto://noah@abjc.tech](noah@abjc.tech)

**Comment by noahkamara** [comment]
@inzi2u Does the issue persist with the latest build?

**Comment by inzi2u** [comment]
with the current build. things are working great. i was able to log in to
the server.

**Comment by noahkamara** [comment]
That's good to know. If you encounter any other issues please create a new issue :)
